gateway/fedimint-gateway-ui/src/mnemonic.rs

```diff
@@ -0,0 +1,87 @@
+use fedimint_gateway_common::MnemonicResponse;
+use maud::{Markup, html};
+
+use crate::DynGatewayApi;
+
+pub async fn render<E>(api: &DynGatewayApi<E>) -> Markup
+where
+    E: std::fmt::Display,
+{
+    let result = api.handle_mnemonic_msg().await;
+
+    match result {
+        Ok(MnemonicResponse { mnemonic, .. }) => {
+            html! {
```

Comment:
Capturing feedback from dev call: perhaps it's better to move the mnemonic into a "Danger Zone" or similar section at the bottom of the dashboard. Can be addressed in a followup.